docs/provider.md

```diff
@@ -60,7 +60,7 @@ new Verifier(opts).verifyProvider().then(function () {
 | `provider`                  | false     | string                         | Name of the provider if fetching from a Broker                                                                                                                                                     |
 | `consumerVersionSelectors`  | false     | ConsumerVersionSelector\|array | Using [Selectors](https://docs.pact.io/pact_broker/advanced_topics/consumer_version_selectors/) is a way we specify which pacticipants and versions we want to use when configuring verifications. |
 | `consumerVersionTags`       | false     | string\|array                  | Retrieve the latest pacts with given tag(s)                                                                                                                                                        |
-| `providerVersionTags`       | false     | string\|array                  | Tag(s) to apply to the provider application                                                                                                                                                        |
+| `providerVersionTags`       | false     | string\|array                  | Tag(s) to apply to the provider application                                   | `providerVersionBranch`       | false     | string                 | Branch to apply to provider application version (recommended to set) |
```

Comment:
I think this needs to be a new line on its own?

Reply:
It does, fail edit in the GH editor